pvlib/irradiance.py

```diff
@@ -2915,3 +2917,61 @@ def dni(ghi, dhi, zenith, clearsky_dni=None, clearsky_tolerance=1.1,
             (zenith < zenith_threshold_for_zero_dni) &
             (dni > max_dni)] = max_dni
     return dni
+
+
+def component_sum_irradiance(zenith,
+                             ghi=None,
+                             dhi=None,
+                             dni=None,
+                             clearsky_dni=None):
+    """
+    Use the component sum equations to calculate the missing series, using
+    the other available time series. One of the three parameters (ghi, dhi,
+    dni) is passed as None, and the other associated series passed are used to
+    calculate the missing series value.
+
+    Parameters
+    ----------
+    zenith : Series
+        True (not refraction-corrected) zenith angles in decimal
```

Comment:
I assume this was taken from the docstring of `irradiance.dni`, but if we're going to comment on refraction, I suspect we actually _do_ want refraction-corrected zenith angles here.  I see that `ModelChain._complete_irradiance` passes `solar_position.zenith` (not `apparent_zenith`).  Is there a good reason for this?  Apparent zenith makes more intuitive sense to me.

Reply:
I originally used zenith as that was what was originally used in the modelchains module. I changed to apparent zenith in https://github.com/pvlib/pvlib-python/pull/1567/commits/b0ae6f4a670b79b314ea11b8a377e9c540471f49